benches/benches/set/alu.rs

```diff
@@ -14,26 +14,13 @@ use fuel_core_types::fuel_asm::wideint::{
     MulArgs,
 };
 
-/// Allocates a byte array from heap and initializes it. Then points `reg` to it.
-fn aloc_bytearray<const S: usize>(reg: u8, v: [u8; S]) -> Vec<Instruction> {
-    let mut ops = vec![op::movi(reg, S as u32), op::aloc(reg)];
-    for (i, b) in v.iter().enumerate() {
-        if *b != 0 {
-            ops.push(op::movi(reg, *b as u32));
-            ops.push(op::sb(RegId::HP, reg, i as u16));
-        }
-    }
-    ops.push(op::move_(reg, RegId::HP));
-    ops
-}
-
-fn make_u128(reg: u8, v: u128) -> Vec<Instruction> {
-    aloc_bytearray(reg, v.to_be_bytes())
-}
+#[path = "../utils.rs"]
```

Comment:
I think you should be able to import `utils` on the `set` module level and be able to use them here from `super`

Reply:
Yes. From the `vm.rs` because that is the parent to `set`.